superset/migrations/versions/2022-07-07_13-00_c747c78868b6_migrating_legacy_treemap.py

```diff
@@ -29,6 +32,14 @@
 
 
 def upgrade():
+    # Ensure `slice.params` and `slice.query_context`` in MySQL is MEDIUMTEXT
+    # before migration, as the migration will save a duplicate form_data backup
+    # which may significantly increase the size of these fields.
+    if isinstance(op.get_bind().dialect, MySQLDialect):
+        # If the columns are already MEDIUMTEXT, this is a no-op
+        op.execute("ALTER TABLE slices MODIFY params MEDIUMTEXT")
```

Comment:
@ktmud why raw SQL here and not the Alembic [alter_column](https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.alter_column) method?

Reply:
Because this is only for one dialect and raw SQL is easier to read.